js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -919,6 +958,15 @@ export const AuctionCard = ({
           to redeem their bids for them right now.
         </h3>
       </MetaplexModal>
+      <CongratulationsModal
```

Comment:
What about having two modal components?
That way you don't need those constants in components.

```suggestion
      <CongratulationsModal isModalVisible={!!shouldOpenPurchaseModal} ...  />
      <CongratulationsModal isModalVisible={!!shouldOpenClaimModal}  buttonText="'View My Items'" ... />
```